modules/ducktests/tests/ignitetest/services/utils/control_utility.py

```diff
@@ -127,6 +131,70 @@ def tx_kill(self, **kwargs):
         res = self.__parse_tx_list(output)
         return res if res else output
 
+    def validate_indexes(self, check_assert: bool = None):
+        """
+        Validate indexes.
+        If check is true, will be return
+        """
+        data = self.__run("--cache validate_indexes")
+
+        if check_assert is not None:
+            assert (('no issues found.' in data) == check_assert), data
+
+    def idle_verify(self, check_assert: bool = None):
+        """
+        Idle verify.
+        """
+        data = self.__run("--cache idle_verify")
+
+        if check_assert is not None:
+            assert (('idle_verify check has finished, no conflicts have been found.' in data) == check_assert), data
+
+    def idle_verify_dump(self, node=None, return_path: bool = False):
+        """
+        Idle verify dump.
+        """
+        data = self.__run("--cache idle_verify --dump", node=node)
+
+        if return_path & ('VisorIdleVerifyDumpTask successfully' in data):
+            match = re.search(r'/.*.txt', data)
+            return match[0]
+
+        return data
+
+    def snapshot_create(self, snapshot_name: str, sync_mode: bool = True, time_out: int = 60):
+        """
+        Create snapshot.
+        """
+        res = self.__run(f"--snapshot create {snapshot_name}")
+
+        self.logger.info(res)
+
+        if ("Command [SNAPSHOT] finished with code: 0" in res) & sync_mode:
+            self.await_snapshot(time_out)
+
+    def await_snapshot(self, time_out=60):
+        """
+        Waiting for the snapshot to complete.
+        """
+        delta_time = datetime.now() + timedelta(seconds=time_out)
+
+        while datetime.now() < delta_time:
+            for node in self._cluster.nodes:
+                mbean = JmxClient(node).find_mbean('snapshot')
+                start_time = int(list(mbean.__getattr__('LastSnapshotStartTime'))[0])
+                end_time = int(list(mbean.__getattr__('LastSnapshotEndTime'))[0])
+                err_msg = list(mbean.__getattr__('LastSnapshotErrorMessage'))[0]
+
+                if (0 < start_time < end_time) & (err_msg == ''):
+                    return
```

Comment:
this snapshot is not related to the created, just a random one finished.
Should we finish https://issues.apache.org/jira/browse/IGNITE-13510 before making this?